index.js

```diff
@@ -1,15 +1,24 @@
-const express = require('express')
-const { getSightings } = require('./utils.js')
-require('dotenv').config()
+const express = require("express");
+const { getSightings } = require("./utils.js");
+var cors = require("cors");
+require("dotenv").config();
 
 const PORT = process.env.PORT;
 const app = express();
 
+app.use(cors());
+
 app.get("/sightings", async (req, res) => {
   const sightings = await getSightings();
   res.json(sightings);
 });
 
+app.get("/sightings/:sightingIndex", async (req, res) => {
+  const sightings = await getSightings();
+
+  res.send(sightings[req.params.sightingIndex]);
```

Comment:
You probably won't be able to access sightingIndex like so. If it works great, if not try this : `req.params["sightingIndex"]`

Reply:
Alternatively you can also destructure. 

`const { sightingIndex } = req.params;`